src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitNotificationService.kt

```diff
@@ -39,10 +41,23 @@ class VisitNotificationService(
     }
   }
 
+  private fun isNotADeleteEvent(nonAssociationChangedNotification: NonAssociationChangedNotificationDto): Boolean {
+    val nonAssociations = prisonerService.getOffenderNonAssociationList(nonAssociationChangedNotification.prisonerNumber)
```

Comment:
what will happen if this call fails for timeout?

Reply:
good point